### PR TITLE
For EmberTimes No. 76: Add Contributors + December Writeup

### PR DESCRIPTION
**Pull request by jessica-jordan**

### CODE REVIEW

File: source/blog/2018-12-07-the-ember-times-issue-76.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+author: Jessica Jordan, the crowd
 tags: Recent Posts, Newsletter, Ember.js Times, Ember Times, 2018
 alias : "blog/2018/12/07-the-ember-times-issue-76.html"
 responsive: true
 ---
 
 εte sεn Emberistas? 🐹
 
-<SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>
+Join with your **contributions** for **December** 🐹🎄,
```
Comment (amyrlam) [inline source/blog/2018-12-07-the-ember-times-issue-76.md]:
```suggestion
Join with your **contributions** for **DecEmber** 🐹🎄,
```

File: source/blog/2018-12-07-the-ember-times-issue-76.md [hunk 2]
```diff
@@ -22,8 +22,18 @@ responsive: true
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [Jingle bells 🔔, jingle bells 🔔, ✨ DecEmber 🐹🎄 all the way!](https://www.emberjs.com/blog/2018/11/29/december-event.html)
```
Comment (amyrlam) [inline source/blog/2018-12-07-the-ember-times-issue-76.md]:
```suggestion
## [Jingle Bells 🔔, Jingle Bells 🔔, ✨ DecEmber 🐹🎄 All the Way!](https://www.emberjs.com/blog/2018/11/29/december-event.html)
```

File: source/blog/2018-12-07-the-ember-times-issue-76.md [hunk 3]
```diff
@@ -22,8 +22,18 @@ responsive: true
...
+
+Find one of the issues 🔍
```
Comment (amyrlam) [inline source/blog/2018-12-07-the-ember-times-issue-76.md]: Do you want these to be on different lines? Not sure how to make that show up at first glance

Comment (jessica-jordan) [reply source/blog/2018-12-07-the-ember-times-issue-76.md]: Yes exactly, thank you for checking! Here's the update :sparkles:

capture d ecran 2018-12-06 a 19 42 30

File: source/blog/2018-12-07-the-ember-times-issue-76.md (hunk 1 above)
Comment (jessica-jordan) [reply source/blog/2018-12-07-the-ember-times-issue-76.md]: 🙇‍♀️

File: source/blog/2018-12-07-the-ember-times-issue-76.md (hunk 2 above)
Comment (jessica-jordan) [reply source/blog/2018-12-07-the-ember-times-issue-76.md]: ❤️

**Pull request merged**